docs/source/catalog.rst

```diff
@@ -287,6 +285,11 @@ The special value ``"catdir"`` implies that cached files will appear in the same
 catalog file in which the data source is defined, within a directory named "intake_cache". These will
 not appear in the cache usage reported by the CLI.
 
+Optionally, the cache section can have a ``regex`` attribute, that modifies the path of the cache on
+the disk. By default, the cache path is made by concatenating ``cache_dir``, dataset name, hash of
+the url, and the url itself (without the protocol). ``regex`` attribute allows to remove part of the
```

Comment:
Is there a better description of regex than this? It feels a little hard to understand - but indeed, the usage *is* hard to uderstand.

Reply:
I was not able to come up with something clear and succinct, so decided to explicitly spell out what it does. 

What could be a good idea, is to warn user about cache files ending up in cwd if regex is not well specified. Where well specified regex to my understanding is one that matches part of the url that includes the beginning, but excludes the protocol. Should I add this?

Reply:
> warn user about cache files ending up in cwd

Sorry, I thought that would no longer be the case, that they would instead end up in the cache directory, under the hash, but with a deeper path

Reply:
> Should I add this?

but yes, I think exactly that wording would be good